server/storeAppointment.php

```diff
@@ -24,7 +24,7 @@ function storeAppointment($data){
 	try {
 		$DB_CONN->beginTransaction();
 
-		$email = '';
+		$email = null;
```

Comment:
For some reason, we were using the empty string for the nullable email field in the DB. I decided to change this to `NULL`